### PR TITLE
DOCS-9779 added links to runtime manager alerts page

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/using-api-alerts.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ An API alert is an alarm that flags one of the following problems:
 * The API returns a specified HTTP error code.
 * The API response time exceeds a certain amount.
 
-Runtime Manager alerts differ in nature from API alerts. Runtime Manager flags CPU usage, secure data gateway up/down, and deployment success/failure, to name a few. API alerts, on the other hand, relate to an API request or response.
+xref:runtime-manager::alerts-on-runtime-manager.adoc[Runtime Manager alerts] differ in nature from API alerts. Runtime Manager flags CPU usage, secure data gateway up/down, and deployment success/failure, to name a few. API alerts, on the other hand, relate to an API request or response.
```
Comment (hannanelson) [inline modules/ROOT/pages/using-api-alerts.adoc]: The topic you're linking to is called "Alerts", so maybe move Runtime Manager out of the link?

Comment (hannanelson) [inline modules/ROOT/pages/using-api-alerts.adoc]:
```suggestion
xref:runtime-manager::alerts-on-runtime-manager.adoc[Alerts] in Runtime Manager differ from API alerts. For example, Runtime Manager flags CPU usage, secure data gateway up and down, and deployment success and failure. API alerts relate to an API request or response.
```

Comment (priyaayyar) [reply modules/ROOT/pages/using-api-alerts.adoc]: Done! Thanks!

Comment (priyaayyar) [reply modules/ROOT/pages/using-api-alerts.adoc]: Done! Thanks!

**Review by hannanelson — APPROVED** [review]

**Pull request merged**